### PR TITLE
OFS-14: Added models and services shell

### PR DESCRIPTION
**Pull request by dborowiecki**
TICKET: [OFS-14](https://openimis.atlassian.net/browse/OFS-14)

### CODE REVIEW

File: contract/services.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        pass
+
+    def __eq__(self, other):
+        return isinstance(other, self.__class__) and self.__dict__ == other.__dict__
+
+    def submit(self, submit):
+        pass
+
+    def amend(self, submit):
+        pass
```
Comment (delcroip) [inline contract/services.py:18]: please add . update, create and delete

File: contract/services.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+@core.comparable
+class Payment(object):
+
+    def __init__(self, policy_holder):
+        self.policy_holder = policy_holder
+        pass
+
+    def __eq__(self, other):
+        return isinstance(other, self.__class__) and self.__dict__ == other.__dict__
+
```
Comment (delcroip) [inline contract/services.py:41]: please add, create, update, delete and asign creditnote

File: contract/services.py (hunk 1 above)
Comment (dborowiecki) [reply contract/services.py:18]: Columns added

File: contract/services.py (hunk 2 above)
Comment (dborowiecki) [reply contract/services.py:50]: Methods added

**Comment by dborowiecki** [comment]
@delcroip I've added requested changes

**Review by delcroip — APPROVED** [review]
ok for me

**Pull request merged**